ResourceStatusSystem/Command/PropagationCommand.py

```diff
@@ -0,0 +1,53 @@
+''' PropagationCommand module
+'''
+
+from DIRAC                                                      import S_OK, S_ERROR
+from DIRAC.ResourceStatusSystem.Client.ResourceStatusClient     import ResourceStatusClient
+from DIRAC.ResourceStatusSystem.Command.Command                 import Command
+from DIRAC.ResourceStatusSystem.Utilities                       import CSHelpers
+
+__RCSID__ = '$Id:  $'
+
+class PropagationCommand( Command ):
+
+  def __init__( self, args = None, clients = None ):
+
+    self.rssClient = ResourceStatusClient()
+    super( PropagationCommand, self ).__init__( args, clients )
+
+  def doNew( self, masterParams = None ):
+     return S_OK()
+
+  def doCache( self ):
+
+    if not self.args['site']:
+      return S_ERROR('site was not found in args')
+
+    site = self.args['site']
+
+    storageElements = CSHelpers.getSiteStorageElements(site)
+    computingElements = CSHelpers.getSiteComputingElements(site)
```

Comment:
There might be other resources types (e.g. catalogs, but also queues, or others) that are part of a site. Can't we just use a generic `getSiteElements` call?